packages/runtime/container-runtime/src/opLifecycle/README.md

```diff
@@ -66,47 +68,30 @@ As `FlushMode.TurnBased` accumulates ops, it is the most vulnerable to run into
 
-## Grouped batching
-
-The `IContainerRuntimeOptions.enableGroupedBatching` option has been added to the container runtime layer and is **off by default**. This option will group all batch messages under a new "grouped" message to be sent to the service. Upon receiving this new "grouped" message, the batch messages will be extracted and given the sequence number of the parent "grouped" message.
-
-The purpose for enabling grouped batching on top of compression is that regular compression won't include the empty messages in the chunks. Thus, if we have batches with many messages (i.e. more than 4k), we will go over the batch size limit just on empty op envelopes alone.
-
-See [below](#how-grouped-batching-works) for an example.
-
-### Risks
+Compressing a batch yields a batch of the same size, but only the first message has any content - the compressed payload containing all the original batch's content.
+The rest of the batch's messages are empty placeholders to reserve sequence numbers for the compressed messages.
 
-This option should **ONLY** be enabled after observing that 99.9% of your application sessions contains these changes (runtime version "2.0.0-internal.7.0.0" or later). Containers created with this option may not open in future versions of the framework.
-
-This option will change a couple of expectations around message structure and runtime layer expectations. Only enable this option after testing
-and verifying that the following expectation changes won't have any effects:
+## Grouped batching
 
--   batch messages observed at the runtime layer will not match messages seen at the loader layer (i.e. grouped form at loader layer, ungrouped form at runtime layer)
--   messages within the same batch will have the same sequence number
--   client sequence numbers on batch messages can only be used to order messages with the same sequenceNumber
--   requires all ops to be processed by runtime layer (version "2.0.0-internal.1.2.0" or later https://github.com/microsoft/FluidFramework/pull/11832)
+With Grouped Batching enabled (it's on by default), all batch messages are combined under a single "grouped" message _before compression_. Upon receiving this new "grouped" message, the batch messages will be extracted, and they each will be given the same sequence number - that of the parent "grouped" message.
 
-Grouped batching may become problematic for batches which contain reentrant ops. This is the case when changes are made to a DDS inside a DDS 'onChanged' event handler. This means that the reentrant op will have a different reference sequence number than the rest of the ops in the batch, resulting in a different view of the state of the data model.
+The purpose for enabling grouped batching before compression is to eliminate the empty placeholder messages in the chunks. Without grouped batching, a batch with many messages (i.e. more than 4k) will go over the batch size limit just on empty op envelopes alone.
```

Comment:
That's a reason. Probably even bigger reason - cheaper (for user and for service), and less likely to hit service throttling triggers (both services)

Reply:
Thanks